mne/decoding/receptive_field.py

```diff
@@ -39,27 +39,33 @@ class ReceptiveField(BaseEstimator):
         float is passed, it will be interpreted as the `alpha` parameter
         to be passed to a Ridge regression model. If `None`, then a Ridge
         regression model with an alpha of 0 will be used.
-    fit_intercept : bool
+    fit_intercept : bool | None
         If True (default), the sample mean is removed before fitting.
-        Ignored if ``estimator`` is a :class:`sklearn.base.BaseEstimator`.
+        If ``estimator`` is a :class:`sklearn.base.BaseEstimator`,
+        this must be None or match ``estimator.fit_intercept``.
```

Comment:
In this case why not enforce it to be None and then just pull from `self.estimator.fit_intercept`?